packages/block-editor/src/components/font-sizes/utils.js

```diff
@@ -62,6 +62,26 @@ export function getFontSizeClass( fontSizeSlug ) {
 		return;
 	}
 
+	// In the past, we used lodash's kebabCase to process slugs.
+	// By doing so, this method also accepted and converted non string values
+	// into strings. Some plugins relied on this behavior.
+	if ( 'string' !== typeof fontSizeSlug ) {
+		fontSizeSlug = String( fontSizeSlug );
+		// eslint-disable-next-line no-console
+		console.warn( 'The font size slug should be a string.' );
+	}
+
+	// In the past, we used lodash's kebabCase to process slugs.
+	// By doing so, this method also stripped special characters
+	// such as the # in "#FFFFF". Some plugins relied on this behavior.
```

Comment:
Looking at this again, what kind of values are we expecting here? 🤔  `#FFFFFF` would be a color value -- that really doesn't make a whole lot of sense for a font size, so we might not want to support that.

Reply:
Happy to update the comment, but I think we still need to provide this conversion: someone may have been doing `$my_font_size`.